### ISSUE TITLE
Is there a save game feature?

### ISSUE THREAD

**Issue by Holdenw4ve** [ZenoArrows/The-Simpsons-Hit-and-Run#29]
Hello, I've been testing the game for a week, and it has been running nicely.
I just have a question about saving the game because I haven't found a functional way, since it just doesn't let me save without crashing.

**Comment by ZenoArrows** [comment]
Are you using the latest version? Also, just to confirm, you're playing on the switch correct?

**Comment by Holdenw4ve** [comment]
I'm sorry but no, I thought I was in the PS Vita thread...

**Comment by P0liak** [comment]
> I'm sorry but no, I thought I was in the PS Vita thread...

You're on both and yeah saving hasn't been implemented yet

**Comment by Holdenw4ve** [comment]
Welp, then I think I'll just wait, thanks for the response :^)

**Comment by pozezombie** [comment]
Has this changed for the psvita simpsons hit and run port i would love to be able to save my progress

**Comment by p0liak** [comment]
> Has this changed for the psvita simpsons hit and run port i would love to be able to save my progress

Of course, but you must have three additional copies of the save file to start a new game now

**Comment by pozezombie** [comment]
What do you mean additional copies everytime i try to save the game an error message saying 
unable to load game press A to continue 
and i do have the latest files

**Comment by p0liak** [comment]
Grab a pc save file and make additional copies, you must have 4 of them, so save1-4

**Comment by pozezombie** [comment]
okay will do

**Comment by pozezombie** [comment]
<img width="363" height="570" alt="Image" src="https://github.com/user-attachments/assets/2f57d880-95f7-443c-9f51-31cc8d733912" />

**Comment by p0liak** [comment]
After that you'll be able to start a new game and overwrite any of the save files

**Comment by pozezombie** [comment]
like this?
and this is under the
 ux0/data/simpsons

**Comment by p0liak** [comment]
Yep

**Comment by pozezombie** [comment]
when i go to overwrite the save by pressing pause and save game it still says the samething should i try uninstalling and reinstalling

**Comment by p0liak** [comment]
Are you using the latest release ?

**Comment by p0liak** [comment]
Oh, you must join the zealous chuck discord server to have the latest

**Comment by p0liak** [comment]
#showcase

**Comment by pozezombie** [comment]
okay will do

**Comment by pozezombie** [comment]
now im getting the error code 
(C2-12828-1)
psp2core-1774802519-0x0000992ccf-eboot.bin.psp2dmp

**Comment by p0liak** [comment]
Uninstall the old vpk, install the latest

**Comment by pozezombie** [comment]
i did

**Comment by p0liak** [comment]
Grab the files from dropbox (!ports) command and try again, make sure you have the right region (ntsc/pal)

**Comment by pozezombie** [comment]
NTSC for america

**Comment by p0liak** [comment]
Yep

**Comment by pozezombie** [comment]
where is this link to the drop box i downloaded the file from the discord MEGA

**Comment by p0liak** [comment]
!ports and send it

**Comment by p0liak** [comment]
You'll trigger a reply from a bot

**Comment by pozezombie** [comment]
p0liak THANK YOU SO MUCH can i leave you a review it works perfectly
